### DISCUSSION TITLE
Error when connecting via Proxy

### DISCUSSION THREAD

**Discussion by GoldenGamerLP** [Minestom/Minestom#1329] [Help] [answered]
Hello there, so currently im trying to connect to Minestom via Velocity, the proxy. Im using Velocity.enable("secret").

Im using unmodified Minecraft 1.19.2 and ViaVersion on the Proxy.

Im using the latest Minestom version via -SNAPSHOT
and im using the latest development build of Velocity

The error: 
![image](https://user-images.githubusercontent.com/65727384/184532264-d81a0e87-c247-474e-9d22-890faa5b209d.png)

**Comment by MrlnHi** [comment]
Does it work if you don't use ViaVersion and just join with the correct version?

**Comment by GoldenGamerLP** [comment, marked as answer]
I found the issue, it has nothing todo with Minestom. It was somehting with ViaVersion and the VelocitySecret

**Reply by MrlnHi** [reply]
Please describe the solution further, if you can

**Reply by GoldenGamerLP** [reply]
The solution is:
1. If youre using VelocitySecret make sure you know when to use " <- these in configs.
2. If youre using ViaVersion try to dont use it, if its not necessary. It can cause many problems, i dont know why.
3. If you are using a cloud like CloudNet make sure to set the logger level to fine otherwise you may not see any errors like i did.
